Tests/FluentAssertions.Json.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -445,6 +446,60 @@ public void When_a_token_is_equal_to_its_string_representation_the_non_equivalen
                 .WithMessage("Expected JSON document not to be equivalent*");
         }
 
+        [Fact]
+        public void When_a_float_is_within_approximation_check_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            var actual = JToken.Parse("{ \"id\": 1.1232 }");
+            var expected = JToken.Parse("{ \"id\": 1.1235 }");
+            const double precision = 1e-3;
```

Comment:
You are more than welcome to inline the two `precision` constants.
We don't think that _every_ value needs to be explicitly defined in the arrange section.